### PR TITLE
Small fixes suggested by pylint

### PR DESCRIPTION
**Pull request by vulder**

### CODE REVIEW

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity decreasing per file
==============================
+ varats/utils/experiment_util.py  -1
+ varats/paper/paper_config_manager.py  -1
+ varats/data/reports/blame_report.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VaRA/VaRA-Tool-Suite/pullRequest?prid=5065700&bid=16668363)

File: varats/experiments/commit_annotation_report.py [hunk 1]
```diff
@@ -43,18 +43,27 @@
...
+    """
     NAME = "PREPARE"
     DESCRIPTION = "Prepares the analysis by annotating the project with the \
         annotation-script of Florian Niederhuber that is provided through \
         prepare.sh."
 
 
 class Extract(Step):  # type: ignore
+    """
+    Extract step to extract a llvm bitcode file(.bc) from the project.
```
Comment (boehmseb) [inline varats/experiments/commit_annotation_report.py]:
```suggestion
    Extract step to extract a llvm bitcode file (.bc) from the project.
```

File: varats/data/reports/blame_report.py [hunk 2]
```diff
@@ -5,8 +5,8 @@
 import typing as tp
 from pathlib import Path
 from collections import defaultdict
-import yaml
 from datetime import datetime
+import yaml
```
Comment (boehmseb) [inline varats/data/reports/blame_report.py]: Why does codacy complain here?

**Review by boehmseb — CHANGES_REQUESTED** [review]

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity decreasing per file
==============================
+ varats/paper/paper_config_manager.py  -1
+ varats/data/reports/blame_report.py  -1
+ varats/utils/experiment_util.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VaRA/VaRA-Tool-Suite/pullRequest?prid=5065700&bid=16668363)

**Pull request merged**